lib/controller.rb

```diff
@@ -75,6 +75,36 @@ class SlowFood < Sinatra::Base
      erb :menu
   end
 
+  get '/owner' do
+    @starter = Dish.all(category: 'starter')
+    @main_course = Dish.all(category: 'main_course')
+    @dessert = Dish.all(category: 'dessert')
+    erb :owner
+  end
+
+  # def test_hello_form_post
+  #     post '/new_dish/', params={:name, :price :category}
+  #
+  #   end
+  # end
+
+  post '/owner' do
+      ownername = params[:name]
```

Comment:
You have an authentication service - Warden. You can use it to authenticate your user and gain access to the helper method `current_user`.